plugins/python-build/bin/python-build

```diff
@@ -1788,9 +1788,12 @@ build_package_verify_py27() {
 build_package_verify_py30() {
   verify_python "${2:-3.0}"
   try_python_module "bz2" "bzip2 lib"
+  try_python_module "curses" "ncurses lib"
+  try_python_module "ctypes" "libffi lib"
   try_python_module "readline" "GNU readline lib"
   verify_python_module "ssl" "OpenSSL lib"
   try_python_module "sqlite3" "SQLite3 lib"
+  try_python_module "tkinter" "Tk toolkit"
```

Comment:
Not sure about Tkinter. We do not require the corresponding dependency in MacOS in https://github.com/pyenv/pyenv/wiki#suggested-build-environment (but that might be because we could previously use it from XCode SDK).
Then, in environments without an X server, there's no point building a GUI library, pulling all its dependencies.